docs/includes/incl-forking-intro.txt

```diff
@@ -0,0 +1,21 @@
+Fork your Aiven service in order to make a copy of the codebase. You can use it to create a development copy of your production, set up a snapshot to analyze an issue or test an upgrade, or create an instance in a different cloud/geographical location/under a different plan 
```

Comment:
Just a minor suggestion, but perhaps "service" or "environment" could be added to "production":

"... create a development copy of your production environment, set up ..."